packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -336,6 +336,7 @@ class App extends client.App {
           items.paste,
           items.separator,
           onClick(items.insertComment, this.props.actions.addComment),
+          onClick(items.insertNode, this.props.actions.showSuggester),
```

Comment:
The same here… And it looks like duplication. Maybe it’s a good idea to implement a menu items factory somewhere in `xod-client`? Something (roughly) like:

`createEditMenu :: [Action] -> Submenu`